webapp/src/components/assetEditor/assetSidebar.tsx

```diff
@@ -189,35 +201,42 @@ class AssetSidebarImpl extends React.Component<AssetSidebarProps, AssetSidebarSt
                     return <div key={el.name} className="asset-editor-sidebar-detail">{`${el.name}: ${el.value}`}</div>
                 })}
             </List>
-            { asset && <List className="asset-editor-sidebar-controls">
-                {canEdit && <Button
+            <List className="asset-editor-sidebar-controls">
+                {asset && canEdit && <Button
                     label={lf("Edit")}
                     title={lf("Edit the selected asset")}
                     ariaLabel={lf("Edit the selected asset")}
                     leftIcon="icon edit"
                     className="asset-editor-button"
                     onClick={this.editAssetHandler} />}
-                <Button
+                {asset && <Button
                     label={lf("Duplicate")}
                     title={lf("Duplicate the selected asset")}
                     ariaLabel={lf("Duplicate the selected asset")}
                     leftIcon="icon copy"
                     className="asset-editor-button"
-                    onClick={this.duplicateAssetHandler} />
-                {canCopy && <Button
+                    onClick={this.duplicateAssetHandler} />}
+                {asset && canCopy && <Button
                     label={lf("Copy")}
                     title={lf("Copy the selected asset to the clipboard")}
                     ariaLabel={lf("Copy the selected asset to the clipboard")}
                     leftIcon="icon paste"
                     className="asset-editor-button"
                     onClick={this.copyAssetHandler} />}
-                {canDelete && <Button
-                    label={lf("Delete Asset")}
```

Comment:
seems like this change is unrelated to this pr

Reply:
Yea, it's slightly unrelated. We felt the delete button needed to be more consistent with the other asset action buttons in order to better distinguish them from the more high-level palette button. Also aesthetically, it looked odd to have a button with no background in the middle of the set of buttons.